01-upload-a-file/python/es1_upload.py

```diff
@@ -0,0 +1,27 @@
+# http://elementalselenium.com/tips/1-upload-a-file
```

Comment:
Make a docstring out of this! Makes more sense and allows additional use (via `<module>.__doc__`).

``` python
"""
Implementation of http://elementalselenium.com/tips/1-upload-a-file
"""
```
Applies to all files of this PR, of course.